tutorials/fiori-ios-scpms-teched19-01/fiori-ios-scpms-teched19-01.md

```diff
@@ -21,155 +23,168 @@ time: 15
 
 [ACCORDION-BEGIN [Step 1: ](Enable SAP Cloud Platform Mobile Services)]
 
-The SAP Cloud Platform SDK for iOS is designed to work seamlessly with a set of services provided by the SAP Cloud Platform that are optimized for communication with mobile devices, known collectively as Mobile Services. These include not only data services, but also features like analytics, push notifications, and app configuration. Before creating your first app, you'll need to ensure that Mobile Services are enabled for your trial account.
+The SAP Cloud Platform SDK for iOS is designed to work seamlessly with a set of services provided by the SAP Cloud Platform that are optimised for communication with mobile devices, known collectively as Mobile Services. These include not only data services, but also features like analytics, push notifications, and app configuration. Before creating your first app, you'll need to ensure that Mobile Services are enabled for your trial account.
 
 [DONE]
 [ACCORDION-END]
 
-[ACCORDION-BEGIN [Step 2: ](Create iOS app using iOS Assistant)]
+[ACCORDION-BEGIN [Step 2: ](Create a Native Application Cloud Configuration)]
 
 With your account created, you can now use the Assistant to create cloud application configurations on the Mobile Services back end, create destinations pointing to data service endpoints, and generate Xcode projects with a built-in data service layer to access the backend via Swift.
 
 For the remainder of this tutorial, you will use the Assistant to generate an Xcode project that accesses a sample backend hosted on the SAP Cloud Platform, then replace the generated user interface with your own, and finally add the code necessary to integrate a Core ML machine learning model to classify product images.
 
-Let's get started by creating a cloud application definition on Mobile Services connecting to a service named `com.sap.edm.sampleservice.v2`. This service is provided as a sample service on SAP Cloud Platform.
+Let's get started by creating a cloud application definition on Mobile Services connecting to a service named **Mobile Sample OData ESPM**. This service is provided as a sample service on SAP Cloud Platform.
 
-1. Open up the **SAP Cloud Platform SDK for iOS Assistant** and click **Create new**.
+1. Go to your **SAP Cloud Platform Mobile Services Trial** account in your web browser.
+  !![Project creation](fiori-ios-scpms-teched19-01.png)
 
-    ![Project creation](fiori-ios-scpms-teched19-04.png)
+2. You will create a new native application cloud configuration on Mobile Services. To do so click on **Native/Hybrid** on the left-hand side and then on **New**.
```

Comment:
"To do so click on **Native/Hybrid**"
or
"To do so choose **Native/Hybrid**"

"...side and then click **New**."